src/sentences/utils.rs

```diff
@@ -49,6 +49,43 @@ pub(crate) fn parse_hms(i: &str) -> IResult<&str, NaiveTime> {
     )(i)
 }
 
+/// The number of milliseconds in a second.
+const MILLISECS_PER_SECOND: i64 = 1000;
+/// The number of milliseconds in a minute.
+const MILLISECS_PER_MINUTE: i64 = 60000;
+/// The number of milliseconds in a hour.
+const MILLISECS_PER_HOUR: i64 = 3600000;
+
+pub(crate) fn parse_duration_hms(i: &str) -> IResult<&str, Duration> {
```

Comment:
From what I see, all values can be negative and we should check this case.

Another thing that would be good to do is avoid using `i64` and only when creating the `Duration` to cast the value to `i64`.

This will give us only 1 place where we should check if we can cast e.g. u32, u64 to i64.